autoplex/data/common/utils.py

```diff
@@ -1003,7 +1004,8 @@ def cur_select(
 
     Notes
     -----
-    This function calculates the descriptor vector for each atom, then performs CUR selection on the resulting vectors.
+    This function calculates the descriptor vector for each atom,
+    then performs CUR selection on the resulting vectors.
 
     Adapted from:
```

Comment:
Use `References` as a header here ?

Reply:
We have adapted some code from the reference. So, would using "adapted" be better?

Reply:
See this previous comment here why Adapted won't work : 
 https://github.com/autoatml/autoplex/pull/203#discussion_r1833916100

Reply:
Ah, I see. I have changed accordingly.